frontend/src/components/auth/teacher-signup/steps/TeacherSignUpOne.tsx

```diff
@@ -35,13 +35,16 @@ const TeacherSignupOne = ({
   const [firstNameError, setFirstNameError] = React.useState(false);
   const [lastNameError, setLastNameError] = React.useState(false);
   const [emailError, setEmailError] = React.useState(false);
+  const [acceptEmail, acceptEmailError] = React.useState(false);
   const [gradesError, setGradesError] = React.useState(false);
   const history = useHistory();
 
   const { value: gradesValues, setValue: setGradesValue } = useCheckboxGroup({
     defaultValue: watch("grades") || [],
   });
 
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Comment:
```suggestion
  const emailRegex = /\S+@\S+\.\S+/;
```
simplified

Reply:
I believe that the simplified regex would accept things like `@@@@.@`. Not sure how strict we want to be with validation though, realistically speaking I don't think people would intentionally type something like this (though I could imagine a typo like `example@@example.com`